content/en/events/guides/migrating_to_new_events_features.md

```diff
@@ -6,35 +6,59 @@ aliases:
+The Events Explorer replaces the Event Stream, one of Datadog's oldest features. The new Event query syntax requires migration of Event monitors, dashboard widgets, and any other features that query Events. 
+
+The introduction of the Events Explorer and its associated changes means that Event features work similarly to other Datadog products. Event query syntax, explorer and analytics views, processing pipelines and other features look like their counterparts in Logs or APM Traces. See [Events Explorer][1] for more details.
 
 ## What this means for your organization
 
 * New events (after migration) are visible only in the [Events Explorer][2] or dashboard widgets and event overlays.
 
 * Older events (before migration) continue to be visible in the [Event Stream][2] for the remainder of their retention period (13 months).
 
-This guide describes how to migrate your organization’s event monitors and dashboard widgets so that you benefit from the new Events features, and so that you're ready when the legacy event stream and storage are sunset. It also provides [a detailed description of what's changed](#details-of-the-changes).
+This guide describes how to migrate your organization’s event monitors and dashboard widgets so that you benefit from the new Events features. Following this guide ensures that you are ready when Datadog retires the legacy event stream and storage. This guide also provides [a detailed description of what's changed](#details-of-the-changes).
 
 ## Migration process
 
-1. **Request early access.** Events Explorer is in private beta. [Contact support][3] to request access for your organization. The support team sends you a notification when Events Explorer is enabled for your organization. You can access the new Explorer and other features through the Events menu in the main navigation, alongside the existing Stream.
+### Datadog migrates data stored in Datadog servers
+<div class="alert alert-info">
+You do not need to take any action for configuration stored directly in Datadog.
+</div>
+
+Datadog is responsible for migrating data stored in its servers. You do not need to take any action for configuration stored directly in Datadog. The changes to Event query syntax affect two Datadog features: monitors and dashboard widgets.
+
+Datadog migrates monitors that query Events by doing the following:
+- For each legacy Event monitor, create a duplicate that uses the new query syntax. Enable the new Event monitor.
+- Add the prefix "legacy" to the legacy Event monitor, and mute it.
+- Update all SLOs and monitor downtimes that reference the legacy Event monitor to point to the new monitor.
+
+When viewing monitors, you can filter which of the two monitor types you would like to see using the **Event Migration** facet. This facet has available values "Event Monitors V1" and "Event Monitors V2."
+
+Datadog directly migrates dashboard widgets that use Event queries. Datadog does not create any new dashboards or widgets.
```

Comment:
Perhaps add more details after the first sentence: This includes Event Stream, Event Timeline widgets and event overlays in other widgets.